.github/workflows/samples.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow runs a `deno check` for npm against sample apps
+name: Deno check
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  samples:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sample:
+          - slack-samples/deno-issue-submission
+          - slack-samples/deno-starter-template
+          - slack-samples/deno-blank-template
```

Comment:
Blank template seems like it might not be a good candidate to test as it's intentionally blank

Reply:
Makes sense, I decided to put it in because it is one of the recommended samples when executing `slack create`

Reply:
Ahh, good thought! I'm fine leaving it as it's not currently erroring. If it ends up giving us difficulties down the line we can always remove it then.